### PR TITLE
Allow render a single fragment

### PR DESCRIPTION
**Pull request by kevinsimper**
I just quickly made some changes that would allow me to render a single fragment related to the issue I opened just before https://github.com/acro5piano/typed-graphqlify/issues/87

This is just a proof of concept :)

### CODE REVIEW

File: src/render.ts [hunk 1]
```diff
@@ -268,3 +268,21 @@ export function render(value: Record<string, unknown>): string {
...
+  }
+
+  // Render main body.
+  renderObject(undefined, value, context)
+
+  // Render the fragments defined in each context until we have no more to render.
+  let currentContext: RenderContext = {
+    // The current context for execution.
+    fragments: new Map(),
+  }
```
Comment (acro5piano) [inline src/render.ts:285]: Can `currentContext` be `const` ?

**Comment by kevinsimper** [comment]
> Could you remove package-lock.json as well cause we totally use Yarn?

Oversaw that, of course 😄

**Comment by acro5piano** [comment]
Thank you for your contribution!

**Comment by acro5piano** [comment]
Was released on `2.4.0`. 
Also feel free to add docs for this feature by creating another PR. Thanks

**Comment by kevinsimper** [comment]
@acro5piano Awesome, thank you! I will add the docs.

I was unsure with if it was actually the best way to do it, the type of fragment made it a bit difficult to use it: https://github.com/acro5piano/typed-graphqlify/blob/master/src/graphqlify.ts#L54

What do you think?

**Comment by acro5piano** [comment]
@kevinsimper 
Thanks!

> I was unsure with if it was actually the best way to do it,

I think this fragment mechanism was created to safely store and iterate all fragments at once.
@luvies knows more than I do ;)

**Pull request merged**